app/build.gradle

```diff
@@ -75,6 +75,12 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutineVersion"
```

Comment:
On `build.gradle` top level we could use [Extra properties](https://docs.gradle.org/current/userguide/writing_build_scripts.html#sec:extra_properties) to centralize all the versions.